src/components/projects-summary/projects-summary.controller.ts

```diff
@@ -136,6 +140,7 @@ export class ProjectsSummaryController implements angular.IController {
     var projects: any = _.toArray(projectData.by('metadata.creationTimestamp'));
     var orderByDate: any = this.$filter('orderObjectsByDate');
     this.ctrl.projects = orderByDate(projects, true);
+    this.ctrl.projects = this.RecentlyViewedProjects.orderByMostRecentlyViewed(this.ctrl.projects);
```

Comment:
Per my comment in the common PR, would just call `orderByMostRecentlyViewed` here and do the fallback ordering by creation date inside of `orderByMostRecentlyViewed`.